protocols/GoNL/BwaAlignAndSortSam.sh

```diff
@@ -44,7 +44,7 @@ if [ "${Library}" == "" ]
 then
 	READGROUPLINE="@RG\tID:${filePrefix}\tPL:illumina\tLB:${filePrefix}\tSM:${externalSampleID}"
 else
-	READGROUPLINE="@RG\tID:${filePrefix}\tPL:illumina\tLB:${Library}\tSM:${externalSampleID}"	
+	READGROUPLINE="@RG\tID:${filePrefix}_${Library}\tPL:illumina\tLB:${Library}\tSM:${externalSampleID}"	
```

Comment:
Change #string Library & ${Library}-> #string library & ${library} First keyword with lowercase.
Where is this parameter declared? I do no see a (test) parameter file for this variable nor a reference to the declaration of this parameter, do you need to still need to commit this file?